libcockatrice_protocol/libcockatrice/protocol/pb/CMakeLists.txt

```diff
@@ -165,13 +165,20 @@ set(PROTO_FILES
     session_event.proto
 )
 
+if(MSVC)
+  set(unused_warning /wd4100)
+else()
+  set(unused_warning -Wno-unused-parameter)
```

Comment:
I just noticed my compilation output for pb has a lot of -Wmissing-declarations instead, I still think we can simply disable -Wall entirely on the generated code.